react-todo-app/src/components/TodoContainer.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from "react"
+import { Route, Switch } from "react-router-dom"
+import About from "../pages/About"
+import NoMatch from "../pages/NoMatch"
+import Header from "./Header"
+import Navbar from "./Navbar"
+import InputTodo from "./InputTodo"
+import TodoList from "./TodoList"
+import { v4 as uuidv4 } from "uuid"
+
+const TodoContainer = () => {
+  const [todos, setTodos] = useState(getInitialTodos())
+
+  // useEffect(() => {
+  //   console.log("test run")
+  //   // getting stored items
+  // const temp = localStorage.getItem("todos")
+  // const loadedTodos = JSON.parse(temp)
+
+  // if (loadedTodos) {
+  //   setTodos(loadedTodos)
+  // }
+  // },[setTodos])
```

Comment:
One suggestion is to delete commented code from the files

Reply:
thanks